docs/faq.rst

```diff
@@ -1,11 +1,32 @@
 FAQs
 ====
 
-How does Mantl compare to `OpenStack Magnum <https://wiki.openstack.org/wiki/Magnum>`_?
+How are Mantl components chosen_?
+--------------------------------------------------------------------------------------
+
+A service design methodology was used to assemble Mantl components. Priority was given
+to sensing the large shifts in this technology and to defining an architecture 
+consistent with mode 1 to mode 2+ methodologies. Once the architecture was optimized 
+feedback was requested from the enterprise developers and SP partners. Popular open source
```

Comment:
That should be clarified here, I am one of the main committers to Mantl and don't know what it is :smile: